tools/api-markdown-documenter/src/api-item-transforms/configuration/Transformations.ts

```diff
@@ -55,138 +57,74 @@ export type TransformApiItemWithoutChildren<TApiItem extends ApiItem> = (
 /**
  * Transformations for generating {@link DocumentationNode} trees from different kinds of API content.
  *
- * @remarks For any transformation not explicitly configured, a default will be used.
- *
  * @public
  */
 export interface ApiItemTransformations {
-	/**
-	 * Generates the default layout used by all default API item transformations.
-	 *
-	 * @remarks
-	 *
-	 * Can be used to uniformly control the default content layout for all API item kinds.
-	 *
-	 * API item kind-specific details are passed in, and can be displayed as desired.
-	 *
-	 * @returns The list of {@link SectionNode}s that comprise the top-level section body for the API item.
-	 */
-	readonly createDefaultLayout: (
-		apiItem: ApiItem,
-		childSections: SectionNode[] | undefined,
-		config: ApiItemTransformationConfiguration,
-	) => SectionNode[];
-
-	/**
-	 * Transformation to generate a {@link SectionNode} for a `Call Signature`.
-	 */
```

Comment:
Should we keep all the existing docs? I see only a few were kept.

Reply:
I kept the couple that had special notes on them. For the others, they seemed redundant. Cases like this always give me pause - I generally prefer docs even in cases like this, but the amount of spatial clutter they create in the file is a bit annoying :/